test/search.spec.js

```diff
@@ -36,7 +36,7 @@ describe('Search', () => {
     let query = (temp.getMonth() + 1) + '-' + temp.getDate();
     request.get('/api/documents?limit=5&date=2016-' + query).set('x-access-token', token).set('Accept', 'application/json').expect(200).end(
```

Comment:
### Nitpick

This line is too long, definitely longer that 80 characters as specified by the airbnb spec.